TestGame/code.dm

```diff
@@ -44,6 +44,13 @@
 		src.client.screen += new /obj/order_test_item/plane_master //used for render tests
 		src.add_client_image()
 
+	verb/winget_test()
+		world.log << "windows: [json_encode(winget(usr, null, "windows"))]"
+		world.log << "panes: [json_encode(winget(usr, null, "panes"))]"
+		world.log << "menus: [json_encode(winget(usr, null, "menus"))]"
+		world.log << "macros: [json_encode(winget(usr, null, "macros"))]"
```

Comment:
Display this to the user instead of the console
```suggestion
		usr << "windows: [json_encode(winget(usr, null, "windows"))]"
		usr << "panes: [json_encode(winget(usr, null, "panes"))]"
		usr << "menus: [json_encode(winget(usr, null, "menus"))]"
		usr << "macros: [json_encode(winget(usr, null, "macros"))]"
```